src/main/service/publicationService.ts

```diff
@@ -2,16 +2,30 @@ import {PublicationRequests} from '../resources/requests/publicationRequests';
 import {Artefact} from '../models/Artefact';
 import {SearchObject} from '../models/searchObject';
 import {ListType} from '../models/listType';
+import sinon from 'sinon';
 
 const listData = require('../resources/listLookup.json');
 const publicationRequests = new PublicationRequests();
 
+sinon.stub(publicationRequests, 'getPubsPerLocation').returns('location,count\n1,2\n3,1\n');
+
 export class PublicationService {
 
   public async getIndividualPublicationMetadata(artefactId, userId: string, admin = false): Promise<any> {
     return publicationRequests.getIndividualPublicationMetadata(artefactId, userId, admin);
   }
 
+  public async getCountsOfPubsPerLocation(): Promise<Map<number, number>> {
+    const response = await publicationRequests.getPubsPerLocation();
+    const splitresp = response.split('\n').slice(1, -1);
```

Comment:
Can we change this to use camel case on the variables currently not using it to help readability.

Reply:
sorted